src/components/Jobs.vue

```diff
@@ -92,7 +96,21 @@ export default {
       });
     },
     runSelectedJob() {
-      console.log('Run job: ' + this.runnableJob.selected.name);
+      let apiPath = this.runnableJob.selected.path;
```

Comment:
@johncrossman Idle curiosity - why `let` here? I know there are subtle effects on scoping but am not used to the idioms.

Reply:
`let` is preferred ES6 usage. See https://vegibit.com/es6-let-vs-var-vs-const/